src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,42 @@
-import React from 'react';
+import React, { Dispatch, SetStateAction, useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  onTodoSelect: Dispatch<SetStateAction<Todo | null>>;
+  selectedUserId: number;
+  selectedTodo: Todo | null;
+};
+
+const defaultUser: User = {
+  id: 0,
+  name: 'Default Username',
+  email: 'Default Email',
+  phone: 'Default Phonenumber',
+};
+
+export const TodoModal: React.FC<Props> = ({
+  onTodoSelect,
+  selectedUserId,
+  selectedTodo,
+}) => {
+  const [user, setUser] = useState<User>(defaultUser);
```

Comment:
You're initializing state with props, which is generally not recommended in React. It can lead to bugs and make your component harder to think about. Refer to the links provided in the checklist for more information.

Reply:
But that was your suggestion to make a default user instead of null...